treon/treon.py

```diff
@@ -46,8 +47,9 @@ def main():
     setup_logging(arguments)
     LOG.info('Executing treon version %s', __version__)
     thread_count = arguments['--threads'] or DEFAULT_THREAD_COUNT
+    html_output = arguments['--html'] or False
```

Comment:
Maybe convert to `bool` i.e. `bool(arguments['--html'])` so that we're sure it's a boolean. 

Let's use `is_html` as the variable name to indicate it's a boolean.